index.php

```diff
@@ -32,15 +32,15 @@
                     <span>We are not an official site, and we are not affiliated with Aternos, Aternos GmbH, exaroton and it's associates and employees.</span>
                 </div>
             </div>
-            <p>Welcome to Aternos Guides, an article site ran by Aternos regulars and an Aternos moderator. Here you may find all kinds of articles about topics and issues that are not covered in Aternos' help center itself. Have an issue that you don't know the fix to? Try our articles below, in the articles page or join the Aternos discord for help.</p>
-
+            <p><b>Welcome to Aternos Guides</b> </p>
+            <p>An article site ran by Aternos regulars and an Aternos moderator. Here you may find all kinds of articles about topics and issues that are not covered in Aternos' help center itself. Have an issue that you don't know the fix to? Try our articles below, in the articles page or join the Aternos discord for help.</p>
```

Comment:
I am still not fond of separating the welcome part from the about part itself. It still feels and looks unnatural.